### PR TITLE
Update symphony components

### PR DESCRIPTION
**Pull request by icewind1991**
Updates the following symphony packages to 2.6.4

- symfony/routing
- symfony/console
- symfony/process
- symfony/event-dispatcher

### CODE REVIEW

File: symfony/console/Symfony/Component/Console/Tests/ApplicationTest.php [hunk 1]
```diff
@@ -45,6 +45,8 @@ public static function setUpBeforeClass()
         require_once self::$fixturesPath.'/Foo5Command.php';
```
Comment (MorrisJobke) [inline symfony/console/Symfony/Component/Console/Tests/ApplicationTest.php]: Can you please drop those unneeded tests and put these folders into the `.gitignore`? Thanks :)

**Comment by PVince81** [comment]
I'm not familiar enough with Symphony to be able to judge this update.

Someone else maybe ? @LukasReschke @nickvergessen

**Comment by icewind1991** [comment]
removed test files

File: composer.lock [hunk 2]
```diff
@@ -958,7 +958,7 @@
                 "reference": "master"
             },
             "type": "library",
-            "time": "2015-01-01 13:17:44"
+            "time": "2014-04-04 10:55:12"
```
Comment (nickvergessen) [inline composer.lock]: This is strange

Comment (nickvergessen) [inline composer.lock]: @icewind1991 any idea, why this was set to a past date?

**Comment by nickvergessen** [comment]
Needs rebasing (new composer.lock)

**Comment by icewind1991** [comment]
rebased

**Comment by nickvergessen** [comment]
something is still in conflict state

**Comment by nickvergessen** [comment]
bump?

**Comment by PVince81** [comment]
please rebase

**Comment by icewind1991** [comment]
rebased

File: symfony/console/Symfony/Component/Console/Tests/ApplicationTest.php (hunk 1 above)
Comment (MorrisJobke) [inline symfony/console/Symfony/Component/Console/Tests/ApplicationTest.php]: @icewind1991 These tests are still in here. Can you drop them too?

Comment (icewind1991) [inline symfony/console/Symfony/Component/Console/Tests/ApplicationTest.php]: done

**Comment by MorrisJobke** [comment]
I tested the console, browsed the web UI, used WebDAV to access, upload and modified files

Works :+1:

**Comment by DeepDiver1975** [comment]
:+1:

**Pull request merged**